File: Queue/queue.c
#include <stdio.h>
#define MAX 50

void enqueue(int);
void dequeue();
void display();

int queue[MAX];
int front = -1;
int rear = -1;

int main() {
    enqueue(18);
    enqueue(42);
    enqueue(53);
    enqueue(20);
    enqueue(90);
    enqueue(9);
    enqueue(34);

    dequeue();

    display();

    return 0;
}

void enqueue(int value) {
    if(rear == MAX-1)
        printf("\nQueue Overflow\n");
    else {
        if(front == -1)
            front=0;
            rear = rear+1;
            queue[rear] = value;
    }
}

void dequeue() {
    if(front == -1 || front > rear)
        printf("\nQueue Underflow\n");
    else {
        front = front+1;
    }
}

void display() {
    if(front==-1)
        printf("Queue is Empty");
    else {
        for(int i = front; i <= rear; i++) {
            printf(" %d ", queue[i]);
        }
    }
}

File: Queue/queue.py
class Queue:
    """ Queue Implementation using Python """

    # Initialization
    def __init__(self):
        self.queue = []

    # Method to Enqueue the element to queue
    def enqueue(self, value):
        return self.queue.insert(0, value)
    
    # Method to dequeue the element to queue
    def dequeue(self):
        if not self.isEmpty():
            return self.queue.pop()
        
    # Method to check if queue is EMPTY
    def isEmpty(self):
        return True if len(self.queue) == 0 else False

    # Method to return the top element of stack
    def peek(self):
        return self.queue[-1]
    
    # Method to print the stack
    def output(self):
        print(self.queue)

# object for queue
object = Queue()

# Enqueue the element
object.enqueue(23)
object.enqueue(56)
object.enqueue(61)
object.enqueue(10)
object.enqueue(7)

# Dequeue the element
object.dequeue()

# Queue output
object.output()